tests/framework/clients/k8s_block.go

```diff
@@ -75,22 +78,23 @@ func (rb *BlockOperation) BlockList() ([]model.BlockImage, error) {
 }
 
 // BlockMap Function to map a Block using Rook
-// Input paramaters -
-//name - path to a yaml file that creates a pod  - pod should be defined to use a pvc that was created earlier
+// Input parameters -
+//name - Pod definition  - pod should be defined to use a pvc that was created earlier
 //mountpath - not used in this impl since mountpath is defined in the pod definition
 //Output  - k8s create pod operation output and/or error
 func (rb *BlockOperation) BlockMap(name string, mountpath string) (string, error) {
-	cmdArgs := []string{name}
-	out, err, status := rb.transportClient.Create(cmdArgs, nil)
-	if status == 0 {
-		return out, nil
+	args := []string{"create", "-f", "-"}
+	result, err := rb.k8sClient.KubectlWithStdin(name, args...)
+	if err != nil {
+		return err.Error(), fmt.Errorf("Unable to map block -- : %s", err)
```

Comment:
Maybe I am not getting this, i get you want the stdout on success, its the whole purpose of the function, what i dont get is why you are returning err.Error(). So on failure you are calling a method to populate a parameter that isnt even going to be read???

return "", fmt.Errorf(....